src/beamlime/applications/data_reduction.py

```diff
@@ -33,22 +33,10 @@ def heatmap_2d(data, threshold=0.2, binning_size=(64, 64)):
 method_map = {"heatmap_2d": heatmap_2d, "handover": lambda x: x}
```

Comment:
I couldn't really write a general handler that can be used for most of `data-reduction` applications yet.
Since now I have clear idea that there should be `data-feeder(pre-data-reduction)`, `data-reduction` and `visualisation`, I should be able to write a general one, but for now they are more or less an example, how to write a `handler` class with the `BeamlimeApplicationInterface`.

So, the `heatmap_2d` is only for this specific one, and didn't intend to use in other handlers.
This `BeamlimeDataReductinoApplication` is one of the `handler` classes that you can use in the configuration.

Maybe I should move those `handler`s to another sub package and keep `applications` only with general ones...

Related to #48